### PR TITLE
Added Ultralytics COCO example & remap_class_ids arg.

### PR DESCRIPTION
**Pull request by MihailMihaylov97**
Added option to pass a dict to remap model class ids to dataset class ids and an example with Ultralytics Yolov8.

### CODE REVIEW

File: examples/stress_test_ultralytics_coco.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        annFile=f"{path}/annotations/instances_val2017.json",
+    )
+    coco_classes = {
+        cat["id"]: cat["name"]
+        for cat in dataset.coco.loadCats(dataset.coco.getCatIds())
+    }
+
+    dataset = [dataset[i] for i in range(5)]
+
+    model = YOLO("yolov8m.pt")
```
Comment (23pointsNorth) [inline examples/stress_test_ultralytics_coco.py:31]: Can we add a comment/link where folks can find the weights?

Comment (VladiGGeorgiev) [reply examples/stress_test_ultralytics_coco.py:31]: The weights are automatically downloading.

Comment (23pointsNorth) [reply examples/stress_test_ultralytics_coco.py:31]: Ah, that's great.

**Review by VladiGGeorgiev — APPROVED** [review]
Look good.

**Pull request merged**